examples/next/docs/js/settings/scripts/test-config.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+const fs = require('fs');
+const { execSync } = require('child_process');
+
+const PROJECT_ROOT = __dirname.split('examples')[0];
+const CODE_SNIPPET_ROOT = __dirname.split('scripts')[0]; // code snippet root is outside the `scripts` directory
+const packageJsonPath = path.join(CODE_SNIPPET_ROOT, 'package.json');
+const packageJsonFile = require(packageJsonPath);
+
+delete packageJsonFile.devDependencies['jasmine']
+delete packageJsonFile.devDependencies['jasmine-console-reporter']
+delete packageJsonFile.devDependencies['puppeteer']
+delete packageJsonFile.devDependencies['http-server']
+
+fs.writeFileSync(packageJsonPath, JSON.stringify(packageJsonFile, null, 2));
+
+// Install `jasmine`, `jasmine-terminal-reporter`, `puppeteer` and `http-server` from local root `node_modules`
+execSync(`npm install --save-dev ${path.join(PROJECT_ROOT, 'node_modules', 'jasmine')}`, {
+  stdio: 'inherit',
+  shell: true
+});
```

Comment:
Why do we do this, is this script necessary? NPM v7 workspaces will not handle the deps for us?

Reply:
@jansiegel ?

Reply:
I've removed this script from all the code examples in https://github.com/handsontable/handsontable/pull/7515/commits/984c0dc2555bede7d4309ff5f7bf4e01c9b0e0ef